lib/plaid/products/link_token.rb

```diff
@@ -0,0 +1,37 @@
+module Plaid
+  # Public: Class used to call the LinkToken sub-product.
+  class LinkToken < BaseProduct
+    def create(configs)
+      post_with_auth 'link/token/create',
+                     CreateResponse,
```

Comment:
please for your sake let's make an array with list of keys and look over it to create the object